quattor/types/component.pan

```diff
@@ -2,6 +2,12 @@ declaration template quattor/types/component;
 
 include 'quattor/functions/validation';
 
+type caf_service_actions = string with choice(
```

Comment:
i'm going to come back on my own remark:
reintroduce the `caf_service_action` type (and remove ti from metaconfig)
and also introduce the following
```pan
type caf_service_actions = {
    daemons ? caf_service_action{}
};
```
that way we can add support by simply including this type in the schema, rather than copy pasting it everywhere